src/ProxyFactory.sol

```diff
@@ -26,7 +26,7 @@ contract ProxyFactory is IProxyFactory {
         return
             address(
                 uint160(
-                    uint(
+                    uint256(
```

Comment:
Any reason for this change? How is it related to 1271 and wouldn't this break compatibility with current deployment?

Reply:
`uint` aliases to `uint256`, but a lot of people say explicitly using `uint256` allows for less ambiguous code :)

Reply:
But why do we need this in this PR? Seems completely unrelated, I'd rather have this within another PR.

Reply:
see initial comment : https://github.com/snapshot-labs/sx-evm/pull/254#issue-2752771234

Reply:
I see, and these uint256 changes were done by the lint script?

Reply:
Some were, but the `uint` was done manually I reckon

Reply:
Everything that is purely formatting is done automatically ; but those that change the code need to be manually done